pkg/aututils/aututils.go

```diff
@@ -133,3 +95,71 @@ func PoolRuleByAvailableCapacity(usage, scalePercentage uint64, expandType strin
 		},
 	}
 }
+
+// VolumeRuleByTotalSize resizes volume by its total size
+func VolumeRuleByTotalSize(capacity int, scalePercentage int, maxSize string) apapi.AutopilotRule {
```

Comment:
line 134 has `PvcRuleBy` and this has `VolumeRuleBy`. For consistency, let's pick one.